articles/native-platforms/ios-swift/10-customizing-lock.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Customizing Lock
+description: This tutorial will show you how to customize the Lock widget UI.
+---
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* CocoaPods 1.0.0
+* XCode 7.3 (7D175)
+* Simulator - iPhone 6 - iOS 9.3 (13E230)
+  :::
+
+<%= include('../../_includes/_package', {
+  pkgRepo: 'native-mobile-samples',
+  pkgBranch: 'master',
+  pkgPath: 'iOS/basic-sample-swift',
+  pkgFilePath: 'iOS/basic-sample-swift/SwiftSample/Info.plist',
+  pkgType: 'replace'
+}) %>
+
+### Before Starting
+
+You need the [Lock](https://github.com/auth0/Lock.iOS-OSX) library integrated in your project. Make sure you've followed the [login tutorial](01-login.md) and you know how to present the login dialog.
+
+### 1. Register your Theme
+
+Lock UI can be customized by creating your own `A0Theme` and overriding the default one anywhere before presenting the login dialog:
+
+```swift
+import Lock
+```
+
+```swift
+let myAwesomeTheme = A0Theme()
+// Configure your theme here (see step 2)
+A0Theme.sharedInstance().registerTheme(myAwesomeTheme)
+```
+
+### 2. Configure your Theme
+
+Lock's widget UI is composed of several parts that can be customized. Here is a list of the available values that you can modify:
+
+![Lock.png](https://github.com/auth0/docs/blob/master/media/articles/libraries/lock-ios/customization/Lock-UI-Parts.png)
```

Comment:
Displays as broken image https://auth0-docs-content-pr-1773.herokuapp.com/docs/quickstart/native/ios-swift/10-customizing-lock

![image](https://cloud.githubusercontent.com/assets/11715799/16798936/16f0286c-48f7-11e6-9efb-ba686418b067.png)


It should be: /media/articles/libraries/lock-ios/customization/Lock-UI-Parts.png